### PR TITLE
Clairify when filename and line number are recorded in Google.Cloud.Diagnostics.*

### LINKED ISSUE #1577: Document exactly when file name and line number can be shown in Google.Cloud.Diagnostics.*
Document exactly when file name and line number can be shown (what debug configurations) with the diagnostics integration libraries.

Some inconsitencies seem to be coming up between netcoreapp1.0 and netcoreapp2.0

cc: @benwulfe

### PR DESCRIPTION
**Pull request by iantalarico**
Fixes #1577

### CODE REVIEW

**Comment by iantalarico** [comment]
@jskeet Friendly ping!

**Comment by jskeet** [comment]
Whoops, sorry - this one had fallen off my radar. Will look now.

File: apis/Google.Cloud.Diagnostics.AspNet/docs/index.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ It allows for simple integration of Stackdriver into ASP.NET applications with m
 
 {{auth}}
 
+# Note
+The Google.Cloud.Diagnostics.AspNet package attempts to collect the filename and line number when
+some entries are collected.  However, to be able to collect this information PDBs must be included with
```
Comment (jskeet) [inline apis/Google.Cloud.Diagnostics.AspNet/docs/index.md]: Not sure what "some entries" means - is there any particular criterion for this? Or is it all entries? (Maybe just remove "some"?)

Fine other than that.

Comment (iantalarico) [reply apis/Google.Cloud.Diagnostics.AspNet/docs/index.md]: no particular criterion, updated. Thanks!

**Pull request merged**